fcrepo-configs/src/main/java/org/fcrepo/config/FedoraPropsConfig.java

```diff
@@ -38,6 +39,7 @@
  * @since 6.0.0
  */
 @Configuration
+@EnableScheduling
```

Comment:
What do you think about adding this to `WebappConfig` instead?

Reply:
I have no particular feelings on this, other than wondering why is this required now?

Reply:
My best guess is that I broke it in this commit: https://github.com/fcrepo/fcrepo/commit/81ac61427376db639b2bc12c83520e70c1ad2ff2

This was the old definition: 

```xml
    <task:scheduler id="taskScheduler" />
    <task:executor id="taskExecutor" pool-size="1" />
    <task:annotation-driven executor="taskExecutor" scheduler="taskScheduler" />
```

It's possible that one of those lines was also enabling scheduling.

Personally, I think it makes more sense for the scheduling to be enabled as part of the webapp config.